backend-services/src/web_asgi/settings.py

```diff
@@ -12,3 +12,8 @@ class AppSettings(Settings):
     staging_mode: bool = False
     wallet_db_name: str
     wallet_db_connection_string: MongoDsn
+    autofund_account: str
```

Comment:
We could make use of `pydantic` validation here to make life easier for ourselves.

```suggestion
    autofund_account: HttpUrl
```

Reply:
you mean for autofund_server right, not autofund_account?

Reply:
apologies yes